test/strongbox/github_api_test.clj

```diff
@@ -193,7 +193,10 @@
                  [{:interface constants/default-interface-version} [:retail]]
                  [{:interface constants/default-interface-version
                    :#interface constants/default-interface-version-classic} [:retail :classic]]
-                 [{:interface 20501 :#interface 90000} [:retail :classic-tbc]]]]
+                 [{:interface 20501 :#interface 90000} [:classic-tbc :retail]]
+
+                 ;; ordering is regular `:interface` first, templated `:#interface` second
```

Comment:
add to todo for next major version: remove support for parsing templated .toc key fields like `## # Interface`